charts/eks-anywhere-packages/templates/certs.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: cert-manager.io/v1
```

Comment:
These files are just copies, right? Might be nice if there was a build step to copy them in rather than duplicate

Reply:
Yea I think that can come after, but they are just copies with templating for the namespace, and annotations.